File: CS/CustomSplashForm.Win/MySplash.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using DevExpress.ExpressApp.Win;

namespace CustomSplashForm.Win {
    public class MySplash : ISplash, ISupportUpdateSplash {
        static private SplashScreenForm form;
        private static bool isStarted = false;
        public void Start() {
            isStarted = true;
            form = new SplashScreenForm();
            form.Show();
            System.Windows.Forms.Application.DoEvents();
        }
        public void Stop() {
            if (form != null) {
                form.Hide();
                form.Close();
                form = null;
            }
            isStarted = false;
        }
        public void SetDisplayText(string displayText) {
        }
        public bool IsStarted {
            get { return isStarted; }
        }
        public void UpdateSplash(string caption, string description, params object[] additionalParams) {
            form.UpdateInfo(description);
        }
    }

}

File: VB/CustomSplashForm.Win/MySplash.vb
Imports System.Linq
Imports System.Text
Imports DevExpress.ExpressApp.Win

Namespace CustomSplashForm.Win
    Public Class MySplash
        Implements ISplash, ISupportUpdateSplash

        Private Shared form As SplashScreenForm
        Private Shared _isStarted As Boolean = False
        Public Sub Start() Implements ISplash.Start
            _isStarted = True
            form = New SplashScreenForm()
            form.Show()
            System.Windows.Forms.Application.DoEvents()
        End Sub
        Public Sub [Stop]() Implements ISplash.Stop
            If form IsNot Nothing Then
                form.Hide()
                form.Close()
                form = Nothing
            End If
            _isStarted = False
        End Sub
        Public Sub SetDisplayText(ByVal displayText As String) Implements ISplash.SetDisplayText
        End Sub
        Public ReadOnly Property IsStarted() As Boolean Implements ISplash.IsStarted
            Get
                Return _isStarted
            End Get
        End Property
        Public Sub UpdateSplash(ByVal caption As String, ByVal description As String, ParamArray ByVal additionalParams() As Object) Implements ISupportUpdateSplash.UpdateSplash
            form.UpdateInfo(description)
        End Sub
    End Class

End Namespace
